exporter/datadogexporter/go.mod

```diff
@@ -3,6 +3,8 @@ module github.com/open-telemetry/opentelemetry-collector-contrib/exporter/datado
 go 1.17
 
 require (
+	github.com/DataDog/agent-payload v4.80.0+incompatible
+	github.com/DataDog/datadog-agent/pkg/quantile v0.31.0-rc.9
```

Comment:
This uses the code from https://github.com/DataDog/datadog-agent/tree/main/pkg/quantile. It should be updated to v0.31.0 when Agent v7.31.0 is released.